requirements/edx/base.txt

```diff
@@ -450,7 +450,7 @@ edx-django-release-util==1.2.0
     #   openedx-blockstore
 edx-django-sites-extensions==4.0.0
     # via -r requirements/edx/base.in
-edx-django-utils==5.2.0
+edx-django-utils==5.3.0
```

Comment:
Hey there, it looks like maybe you bumped this by hand? When I run `make compile-requirements` on master I'm getting some additional downstream changes in development.txt and testing.txt that seem to stem from this commit. In the future could you follow these instructions? https://github.com/openedx/edx-platform/blob/master/requirements/README.rst#upgradingdowngrading-just-one-dependency -- it helps ensure that dependencies stay synchronized and consistent.